contributing/source-code.txt

```diff
@@ -27,20 +27,6 @@ The Bio-Formats repository is available from:
 -  https://github.com/openmicroscopy/bioformats
 -  git://openmicroscopy.org/bioformats.git
 
-Other repositories
```

Comment:
We still need some sort of sub-heading left here, otherwise the network members lists are all under 'Bio-Formats'. I'd reinstate this heading and just remove 'After that,' from the beginning of the first sentence below.